tests/unit/maze_transformer/generation/test_tokenizers.py

```diff
@@ -1,5 +1,170 @@
 """
 Test that the wrapped tokenizer loaded provides the same outputs
 as the original tokenizer (i.e. just using the token map in cfg)
+
+We may want a separate set of tests for different tokenization schemes
 """
-# TODO - Pending tokenizer
+import torch
+from pytest import mark, param
+from transformer_lens import HookedTransformer, HookedTransformerConfig
+
+from maze_transformer.training.config import ConfigHolder
+from maze_transformer.training.mazedataset import MazeDatasetConfig
+from maze_transformer.training.tokenizer import HuggingMazeTokenizer
+from scripts.create_dataset import generate_MazeTokenizer
+
+
+def test_tokenization_encoding():
+    # Check that wrapped tokenizer __call__ returns the same as original tokenizer
+    maze = generate_MazeTokenizer(None, 3)
+
+    # Need to generate a config to extract the token map >.<
+    # TODO: Part of https://github.com/AISC-understanding-search/maze-transformer/issues/77
+    cfg = MazeDatasetConfig(name="testing_maze", grid_n=3, n_mazes=1)
+    node_token_map = cfg.node_token_map
+
+    # Adjacency List Tokenization
+    maze_str_tokens = maze.as_tokens(node_token_map)
+
+    # Manual Tokenization
+    vocab_map = {token: i for i, token in enumerate(cfg.token_arr)}
```

Comment:
Sometimes a variable name with a type in it (Hungarian notation) can be renamed to be even more informative, e.g. instead of vocab_map it could be vocab_to_index, which gives you information about the value as well as the key